docs/source/user-guide/vision/data-classes/DetectionData.rst

```diff
@@ -22,31 +20,34 @@ Accepted Label Format
 ---------------------
 Deepchecks' checks use the :func:`~deepchecks.vision.DetectionData.batch_to_labels` function in order to get the labels in the correct format.
 The accepted label format for is a a list of length N containing tensors of shape (B, 5), where N is the number
-of samples, B is the number of bounding boxes in the sample and each bounding box is represented by 5 values:
-``(class_id, x, y, w, h)``.
+of samples within a batch, B is the number of bounding boxes in the sample and each bounding box is represented by 5 values:
+``(class_id, x_min, y_min, w, h)``.
 
-    x and y are the coordinates (in pixels) of the upper left corner of the bounding box, w
+    x_min and y_min are the coordinates (in pixels) of the **lower left corner** of the bounding box, w
     and h are the width and height of the bounding box (in pixels) and class_id is the class id of the prediction.
 
 For example, for a sample with 2 bounding boxes, the label format may be:
-``[(1, 8.4, 50.2, 100, 100), (5, 26.4, 10.1, 20, 40)]``.
+``tensor([[1, 8.4, 50.2, 100, 100], [5, 26.4, 10.1, 20, 40]])``.
 
 Accepted Prediction Format
 --------------------------
 Deepchecks' checks use the :func:`~deepchecks.vision.DetectionData.infer_on_batch` function in order to get the predictions of the model in the correct format.
 The accepted prediction format is a list of length N containing tensors of shape (B, 6), where N is the number
 of images, B is the number of bounding boxes detected in the sample and each bounding box is represented by 6
-values: ``[x, y, w, h, confidence, class_id]``.
+values: ``[x_min, y_min, w, h, confidence, class_id]``.
 
-    x and y are the coordinates (in pixels) of the upper left corner
-    of the bounding box, w and h are the width and height of the bounding box (in pixels), confidence is the
-    confidence of the model and class_id is the class id.
+    x_min,y_min,w and h represent the bounding box location as before, confidence is the confidence score given by the model to
```

Comment:
Not sure what I think, but consider writing explicitly and not "as before" - what if someone just wants to look at this and not at the label section?

Reply:
Think it is the same as in image format that refers to a different page for details.

As above will be better?

Reply:
Yeah better IMO